### PR TITLE
Mention missing css absolute `@import uri` support in docs

### PR DESCRIPTION
**Pull request by wbinnssmith**
This adds a note to the css section of the Turbopack docs that `@import` of absolute uris is not yet supported and suggests using a link tag as an alternative.

### CODE REVIEW

File: docs/pages/pack/docs/features/css.mdx [hunk 1]
```diff
@@ -50,6 +52,11 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
+<Callout type="info">
+  Turbopack does not currently support absolute uri imports in css, such as `@import(https://example.com/stylesheet.css)`. As a workaround,
+  [add `<link href="https://example.com/stylesheet.css"></link>` to `pages/_document.js`](https://nextjs.org/docs/messages/no-page-custom-font#possible-ways-to-fix-it).
+</Callout>
```
Comment (padmaia) [inline docs/pages/pack/docs/features/css.mdx:58]: This is Next.js specific advice, right? Maybe "To work around this in Next.js, " instead? Also did I hear this advice only works in pages?

**Review by sokra — APPROVED** [review]

File: docs/pages/pack/docs/features/css.mdx [hunk 2]
```diff
@@ -50,6 +52,12 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
+<Callout type="info">
+  Turbopack does not currently support absolute uri imports in css, such as `@import(https://example.com/stylesheet.css)`. As a workaround, add
```
Comment (alexkirsz) [inline docs/pages/pack/docs/features/css.mdx:56]:
```suggestion
  Turbopack does not currently support absolute URI imports in css, such as `@import(https://example.com/stylesheet.css)`. As a workaround, add
```

Comment (alexkirsz) [reply docs/pages/pack/docs/features/css.mdx:56]: Isn't the syntax `@import url("https://example.com/stylesheet.css");`?

File: docs/pages/pack/docs/features/css.mdx [hunk 3]
```diff
@@ -50,6 +52,12 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
...
+  `<link rel="stylesheet" href="https://example.com/stylesheet.css"></link>` to your HTML document. Next.js users using `pages/` should
+  [add this to `pages/_document.js`](https://nextjs.org/docs/messages/no-page-custom-font#possible-ways-to-fix-it).
```
Comment (alexkirsz) [inline docs/pages/pack/docs/features/css.mdx:58]: Next.js 13 also has `app/layout.js`: https://beta.nextjs.org/docs/routing/pages-and-layouts#root-layout-required

I'm not sure whether we support this yet.

**Review by alexkirsz — CHANGES_REQUESTED** [review]

File: docs/pages/pack/docs/features/css.mdx [hunk 4]
```diff
@@ -50,6 +52,12 @@ Using the CSS `@import` syntax to import other CSS files works **out-of-the-box*
 @import './dark.css';
 ```
 
+<Callout type="info">
+  Turbopack does not currently support absolute URI imports in css, such as `@import "https://example.com/stylesheet.css"`. As a workaround, add
+  `<link rel="stylesheet" href="https://example.com/stylesheet.css"></link>` to your HTML document. Next.js users using `pages/` should
```
Comment (alexkirsz) [inline docs/pages/pack/docs/features/css.mdx:57]:
```suggestion
  `<link rel="stylesheet" href="https://example.com/stylesheet.css" />` to your HTML document. Next.js users using `pages/` should
```

**Review by alexkirsz — APPROVED** [review]

**Review by jaredpalmer — CHANGES_REQUESTED** [review]
Need to update for next 13 layout

**Pull request merged**